packages/react-events/drag.js

```diff
@@ -9,6 +9,4 @@
 
 'use strict';
 
-const Drag = require('./src/Drag');
-
-module.exports = Drag.default || Drag;
+module.exports = require('./src/Drag');
```

Comment:
I would prefer to not use such bridges modules. I use rollup for building production app. Such modules require explicitly define `namedExports: { 'react-events/drag': ['Drag', 'useDrag'] }`.

Reply:
We'll get to that if this ever becomes public, but this work is currently limited to experimentation at Facebook.